backend/README.md

```diff
@@ -8,6 +8,71 @@ Add dependencies with `poetry add`
 You can run it with:
 `poetry run uvicorn main:app`
 
+## Installation
+
+1. Clone this repository to your local machine:
+
+   ```bash
+   git clone https://github.com/facebookresearch/MultiModalExplorer.git`
+   ```
+
+2. Navigate to the project directory:
+
+   ```bash
+   cd MultiModalExplorer/backend/multimodalexplorer`
+   ```
+
+3. Create a virtual environment (optional but recommended):
+
+   ```bash
+   python -m venv venv
+   ```
+
+4. Activate the virtual environment (skip this step if you didn't create a virtual environment):
+
+   On Windows:
+
+   ```bash
+   venv\Scripts\activate
+   ```
+
+   On macOS and Linux:
+
+   ```bash
+   source venv/bin/activate
+   ```
```

Comment:
poetry should do all of this for you:

https://python-poetry.org/docs/basic-usage/#using-your-virtual-environment